spec/helpers.rb

```diff
@@ -30,13 +30,13 @@ def get_attributes_by_row(type, file, rownum)
   end
 
   def get_doc(converter)
-    Nokogiri::XML(converter.convert, nil, 'UTF-8').remove_namespaces!
+    Nokogiri::XML(converter.convert, nil, 'UTF-8'){ |c| c.noblanks }.remove_namespaces!
```

Comment:
Creating the Nokogiri::XML object with the noblanks config removes XML nodes with name='text' and no content, so that weird differences don't cause tests to fail.